src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArr = sourceString.split(';');
+  const newStyles = {};
+
+  styleArr.map((style) => {
+    if (style.trim() === '') {
+      return;
+    }
+
+    let [property, value] = style.split(':');
+
+    if (!property || !value) {
+      return;
+    }
+
+    property = property.trim();
+    value = value.trim();
+
+    newStyles[property] = value;
+  });
```

Comment:
[CODE KNOWLEDGE] - The `map` method is intended for transformations and should return a new array. Since you're not using the return value of `map`, you should use `forEach` instead, which is intended for side effects on each element, like populating `newStyles`.